### ISSUE TITLE
Rewrite all the things to use bird2?

### ISSUE THREAD

**Issue by knorrie** [knorrie/network-examples#16]
Reaction time!

We have bird2 now.

Should I rewrite all of this to use bird 2 instead of bird 1?

I can imagine that anyone who wants to start learning bird would immediately want to use bird 2 nowadays.

Is that true?

The biggest change to make is merging the IPv4 and IPv6 configs of course. It's not difficult, it's just a lot of boring work.

**Comment by bastelfreak** [comment]
Yes!
* bird1 is quite old and I don't think it will new features, only bugfixes?
* every modern platform has bird2 available

so I think it's sane now to switch to bird2

**Comment by GoetzGoerisch** [comment]
Yes please @knorrie!

**Comment by knorrie** [comment]
Yes! It's about time. Well, it already was, of course. In the meantime, I've already been using bird2 often enough to easily be able to cover all examples that are in here now already.

So, I guess what I'll be doing is work through all of it myself again, using a Debian 12 base system, convert everything on the go to bird2, and share the work-in-progress in the 'bird2' branch all the time. 

Yesterday, I made the first step to create a new 'lxcbird' Debian 12 virtual machine thing for this in my home network, and then also already made a little bit of a start.

Converting the bird1 stuff to bird2 is pretty straightforward, I guess. I suspect I don't even have to fire up dia to make changes to any of the pictures.

I really think I should *first* just try to get exactly that done (bird 1->2), and resist all other temptations that are screaming at me about all the stuff that could also be improved.

And finally, I have to think about a way of organizing all of this so that it actually gets done, eventually. I think that's what people call 'planning' (brrrrr). I'm not sure about that part yet. But... if I try to do it in the way I just described it should be possible to just time-box the steps.

Every time I see something and think "Oh, silly Knorrie, that's how you were doing things in 2015", I should probably make a note on some todo list, and then NOT actually do anything about it.

**Comment by GoetzGoerisch** [comment]
Thank you for your update.

I suggest that you make a ToDo entry at the places where you would update further.

Looking forward to this.
